python/nav/portadmin/snmputils.py

```diff
@@ -810,6 +814,74 @@ def write_mem(self):
         ])
 
 
+class Dell(SNMPHandler):
+    """Dell INC handler
+
+    Uses DNOS-SWITCHING-MIB
+    """
+
+    from nav.smidumps.dnos_switching_mib import MIB as mib
+
+    switch_port_modes = {
```

Comment:
I would maybe just use constants here, rather than strings. It's a lot easier to detect incorrect variable names than it is to detect random strings with typos...